app/assets/scss/_brief_submission.scss

```diff
@@ -1,3 +1,20 @@
+.instruction-list-item {
```

Comment:
@idavidmcdonald and I talked to @tombye about this and we agreed that (a) yes, your point is a good one and you are right and also that (b) we want to merge this now and fix it later.

Currently, this pattern is being used only twice (soon to be three times).  
- It's being used correctly on the download agreement page (ie, by using the toolkit macro)
- It _will be_ used correctly on the new "signing your agreement" flow
- And it's also used incorrectly on the brief overview/grandparent page where we're using instruction-list classnames but then overriding CSS rules.

As you've pointed out, overriding the general styles in our apps is contrary to what the toolkit is meant to be for and we shouldn't be doing this.

However, having acknowledged that the pattern isn't really working in a general sense, the three times we're using it are all currently working fine and we're not doing anything to make it worse in this pull request.  Breaking the pattern into something more reusable and modular is definitely the going-forward solution, but at this point it just doesn't seem that important.  It's also not entirely clear what breaking down the pattern means in terms of actual code, so it seems like it would require a lot of time to fix properly something that's, frankly, just not a priority right now.

As a side note, the [story](https://www.pivotaltracker.com/story/show/121577345) that David and I are working on has been going for about 2 weeks and has meant writing (and rewriting) a bunch of changes to 4 different repos so far.  Just as totally flawed and selfish humans, we really don't want to be hung up on this particular issue.